File: CTDK_Python/011/_011.py
from math import sqrt
import math

x1 = float(input("Nhao x1: "))
y1 = float(input("Nhao y1: "))
x2 = float(input("Nhao x2: "))
y2 = float(input("Nhao y2: "))
x3 = float(input("Nhao x3: "))
y3 = float(input("Nhao y3: "))

a = sqrt((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1))
b = sqrt((x3 - x2) * (x3 - x2) + (y3 - y2) * (y3 - y2))
c = sqrt((x3 - x1) * (x3 - x1) + (y3 - y1) * (y3 - y1))

p = (a + b + c) / 2
s = sqrt(p * (p - a) * (p - b) * (p - c))
print("Dien tich tam giac: ", s)

File: CTDK/011/011.cpp
#include <iostream>
using namespace std;

int main()
{
	float x1;
	cout << "Nhap x1: ";
	cin >> x1;

	float y1;
	cout << "Nhap y1: ";
	cin >> y1;

	float x2;
	cout << "Nhap x2: ";
	cin >> x2;

	float y2;
	cout << "Nhap y2: ";
	cin >> y2;

	float x3;
	cout << "Nhap x3: ";
	cin >> x3;

	float y3;
	cout << "Nhap y3: ";
	cin >> y3;

	float AB = sqrt((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1));
	float AC = sqrt((x3 - x1) * (x3 - x1) + (y3 - y1) * (y3 - y1));
	float BC = sqrt((x3 - x2) * (x3 - x2) + (y3 - y2) * (y3 - y2));
	float p = (AB + AC + BC) / 2;
	float s = sqrt(p * (p - AB) * (p - AC) * (p - BC));
	cout << "Chu vi tam giac la: " << s << endl;
	return 0;
}